### PR TITLE
Quality Of Life Update

### PR DESCRIPTION
**Pull request by XstreamSpeed**
**Please describe the changes this PR makes and why it should be merged:**
This PR brings many new changes, have a look for better understanding.

**Status and versioning classification:**

### CODE REVIEW

**Comment by XstreamSpeed** [comment]
In the list of commits you can see that the 2 commits that you requested to be changed are still there, but the last 2 commits down at the bottom of the list show that they have been changed, said this to avoid confusion.

File: commands/slash/remove.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const command = new SlashCommand()
       const joinEmbed = new MessageEmbed()
         .setColor(client.config.embedColor)
         .setDescription(
-          "You have to join voice channel first before you can use this command"
+          "You have to a join voice channel before you can use this command"
```
Comment (ManfredRichthofen) [inline commands/slash/remove.js:29]: I think you meant to switch the a and join 
"You have to join a voice channel before you can use this command"

**Comment by DarrenOfficial** [comment]
Please stop making the same pull request with minor tweaks (https://github.com/SudhanPlayz/Discord-MusicBot/pull/839)

Instead put all of your "requested changes" commit into the same pr. you can do so by committing it to whatever branch you're working on and the changes should be updated automatically to the pr you're making.

Thanks.

**Comment by XstreamSpeed** [comment]
Alr

**Pull request closed without merging**